### PR TITLE
Deprecation notice for imageDrawImage()

### PR DESCRIPTION
**Pull request by shaedrich**

### CODE REVIEW

**Comment by KamasamaK** [comment]
What is the meaning of `maximum_version`? We have `deprecated` and `removed` for their respective purposes.

File: data/en/imagedrawimage.json [hunk 1]
```diff
@@ -12,8 +12,8 @@
 		{"name": "y", "description": "", "required": true, "default": "", "type": "numeric", "values": []}
 	],
 	"engines": {
-		"lucee": {"minimum_version": "", "notes": "", "docs": "http://docs.lucee.org/reference/functions/imagedrawimage.html"}
+		"lucee": {"minimum_version": "4.5", "maximum_version": "5", "notes": "This function is deprecated, use ImagePaste instead.", "docs": "http://docs.lucee.org/reference/functions/imagedrawimage.html"}
```
Comment (KamasamaK) [inline data/en/imagedrawimage.json]: What is the meaning of `maximum_version`? We have `deprecated` and `removed` for their respective purposes.

Comment (pfreitag) [reply data/en/imagedrawimage.json]: Yup should probably be deprecated="5" right?

Comment (shaedrich) [reply data/en/imagedrawimage.json]: Mea culpa! Of course!

**Pull request merged**